Modelica/Blocks/Tables.mo

```diff
@@ -427,7 +433,13 @@ extrapolation = 1: Hold the first or last value of the table,
                    (If smoothness is LinearSegments or ConstantSegments
                    this means to extrapolate linearly through the first/last
                    two table points.).
-              = 3: Periodically repeat the table data (periodical function).
+              = 3: Periodically repeat the table data (periodical function), 
+                   where the repetition period is table[end,1]-table[1,1].
+                   For u = table[end,1] the result is table[end,2]. However for the
+                   limit of u downwards to table[end,1] the result is table[<strong>1</strong>,2].
+                   This implies that for correct reproduction of a periodic function
+                   the first and last data values should be the same 
+                   (table[1,2] = table[end,2]).
```

Comment:
```suggestion
                   where the repetition period is table[end,1]-table[1,1].
                   For u = table[end,1] the result is table[end,2:end]. However for the
                   limit of u downwards to table[end,1] the result is table[<strong>1</strong>,2:end].
                   This implies that for correct reproduction of a periodic function
                   the first and last data values should be the same 
                   (table[1,2:end] = table[end,2:end]).
```
The table is designed with the option to have multiple outputs. Either just say that in the text or add these changes to make it clear in code.

Reply:
I believe this is incorrect.
For the limit of a single u (u being any of the inputs) downwards to table[end,1] the result is a single output value table[1,2], not the array table[1,2:end].

Reply:
For the 1Ds table this would be almost correct: one input possibly outputs an array (added suggestion above). For the CombiTable1Dv in this code section, there are as many outputs 'y' as there are inputs 'u', specified by parameter 'columns'. This complicates the discussion, since we must specify if we're talking about a single 'u' from the array, or the whole input array (which will not be equal to table[end,1] all at the same time).
```suggestion
                   where the repetition period is table[end,1]-table[1,1].
                   For input u[i] = table[end,1] the output value is table[end,columns[i]]. 
                   However for the limit of u downwards to table[end,1] the result is 
                   table[<strong>1</strong>,columns[i]].
                   This implies that for correct reproduction of a periodic function
                   the first and last data rows should be the same 
                   (table[1,2:end] = table[end,2:end]).
```